app/screens/Map.js

```diff
@@ -102,7 +102,7 @@ class Map extends Component {
     async componentDidMount(){
         this.props.getCurrentLocation()
         await Font.loadAsync({'MaterialIcons': require('@expo/vector-icons/fonts/MaterialIcons.ttf')})
```

Comment:
This seems psycho, but
`await Font.loadAsync({'MaterialIcons': require('@expo/vector-icons/fonts/MaterialIcons.ttf')})`
works for Android but not iOS. And
`await Font.loadAsync({'Material Icons': require('@expo/vector-icons/fonts/MaterialIcons.ttf')})`
works with iOS but not Android.

Pretty cool. Including both lines makes it work for both platforms. I'm not sure if there's any downside to that.

Reply:
That seems really weird. If it's working, sounds fine for now. I hope there are cleaner ways to achieve this that we can eventually explore.